packages/story-editor/src/masks/index.js

```diff
@@ -43,3 +48,34 @@ function getDefaultElementMask(type) {
   const { isMaskable } = getDefinitionForType(type);
   return isMaskable ? DEFAULT_MASK : null;
 }
+
+const BORDER_MULTIPLIER = 3;
+export function getBorderedMaskProperties(
```

Comment:
Would be great to leave a general comment somewhere explaining how this works.

There's a slight difference currently with "normal" borders -- the frame line is shown in the middle of the border as opposed to around the origin element. Probably this could be unified in #5325 where all the border will be included inside the frame for all elements?
<img width="243" alt="Screenshot 2022-01-24 at 11 57 32" src="https://user-images.githubusercontent.com/3294597/150761812-c5b79269-d5e6-4f0f-9349-3d0b54462371.png">

Reply:
Oh, the middle-aligned border (rather than the outer border) is a significant change, but it's the only way this could work. It changes workflow quite a bit for end-users and is definitely undesirable.

But there is no other way to do this, as it is way too hard to synchronize the two SVG's on top of each other if we want to "punch out" the shape of the element "inside" the border. It just didn't work reliably enough to be useful.

So this is what we have. It's this or nothing, and I feel that this is better.

Reply:
Ah, yes, definitely, just wanted to confirm mainly if this will still work for #5325 as well (where the frame would be moved to outside line of the border) -- to make the border clickable, too.

Reply:
Added documentation in 4193c5d1b.